src/app/Http/Requests/UserUpdateCrudRequest.php

```diff
@@ -24,10 +24,13 @@ public function authorize()
      */
     public function rules()
     {
+        $userModel = config('backpack.permissionmanager.models.user');
+        $userModel = new $userModel();
+        
         $id = $this->get('id') ?? request()->route('id');
 
         return [
-            'email'    => 'required|unique:'.config('permission.table_names.users', 'users').',email,'.$id,
+            'email'    => 'required|unique:'.$userModel->getTable().',email,'.$id,
```

Comment:
Why instantiate a class to get the table name? You can use a configuration key for this.

Reply:
I get your previous PR now. I think it's better than this one. 

I will be closing this as yours do the same but uses the config directly.